src/melee/ft/fighter.h

```diff
@@ -6,18 +6,42 @@
 
 #include "sysdolphin/baselib/gobj.h"
 
-typedef struct { float x, y;    } Vec2;
-typedef Vec Vec3;
+// Points to data in PiCo.dat
+typedef struct _ftCommonData {
+  u8 data_filler_1[0x480];
```

Comment:
if you're reindenting this file, would you mind using 4 spaces like the others?

Reply:
i would not mind, I only did that because it was what others were doing in that file